frontend/packages/dev-console/integration-tests/utilities/appFunctions.ts

```diff
@@ -0,0 +1,99 @@
+import { $, browser, element, by, ExpectedConditions as EC } from 'protractor';
+import { click, elementByDataTestID } from './elementInteractions';
+
+export const ELEMENT_WAIT = 15000;
+
+export enum NavigationMenu {
+  Add = '+Add',
+  Topology = ' Topology',
+  Monitoring = 'Monitoring',
+  Builds = 'Builds',
+  Search = 'Search',
+  Helm = 'Helm',
+  ProjectDetails = 'Project Details',
+  ProjectAccess = 'Project Access',
+  Pipelines = 'Pipelines',
+}
+
+export enum Perspective {
+  Developer = 'Developer Perspective',
+  Administrator = ' Administrator Perspective',
+}
+
+export const naviagteTo = async function(opt: NavigationMenu) {
```

Comment:
New nav changes will break this :(

Needs updating.

Reply:
Yeah I understand, but I want them to keep at one place. Which will be easy for maintenance

Reply:
What i'm saying is that the navigation has changed.
Therefore parts of this function are now invalid.

Reply:
Rebase and re-run @makambalaji You'll see the nav changes and your tests should have issues getting the items (if it actually broke and didn't keep the data-test-ids).

Reply:
Oh okay , got it will check and update it